packages/features/sales-escrows/modals/TransferTokens.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { sendTransaction, useTokensContext } from '@dapp/features-tokens-provider';
+import { AuthContext } from '@dapp/features-authentication';
+import { Container, Flex, HR, Modal, TextInput, Select, Button } from '@origyn-sa/origyn-art-ui';
+import { LinearProgress } from '@mui/material';
+
+const TransferTokensModal = ({ currentToken, open, handleClose }: any) => {
+  const { actor, ogyActor, principal } = React.useContext(AuthContext);
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [token, setToken] = React.useState('OGY');
+  const { tokens, refreshAllBalances } = useTokensContext();
+  const [selectedToken, setSelectedToken] = useState<any>('OGY');
+  const [amount, setAmount] = useState<any>(0);
+  const [receiver, setReceiver] = useState<any>();
+  const [memo, setMemo] = useState<any>();
+  const walletType = localStorage.getItem('loggedIn');
+  const [switchTransfer, setSwitchTransfer] = useState(false);
+
+  const totalAmount = Number(amount) + 0.002
```

Comment:
Number(amount) + tokens[selectedToken].fee and this should not be on the function root level